opal/mca/btl/uct/btl_uct_endpoint.c

```diff
@@ -101,6 +102,28 @@ static void mca_btl_uct_process_modex (mca_btl_uct_module_t *uct_btl, unsigned c
     }
 }
 
+static inline ucs_status_t mca_btl_uct_ep_create_connected_compat (uct_iface_h iface, uct_device_addr_t *device_addr,
+                                                                   uct_iface_addr_t *iface_addr, uct_ep_h *uct_ep)
+{
+#if UCT_API >= UCT_VERSION(1, 6)
```

Comment:
what about check if uct_ep_create_connected is available using autotools/configure? this is standard way to check if some API available.
for example look at constructions AC_CHECK_DECLS() at config/ompi_check_ucx.m4

Reply:
We talked about this and decided for a generic version test -- which, you're right, is unusual / not the preferred way for `configure`-based tests.

The reason is that the UCT API is not guaranteed to have stability over time.  As I understand it, UCT is an internal API that may change at any time without any warning.  So technically, if we went the "usual" way of `configure` tests, we'd have to test for every single UCT API symbol (including parameters) that is used in this BTL.  

That's why we compromised on the version check, instead: this BTL is known to work with UCX versions X through Y.